app/src/main/res/layout-sw600dp-port/submitted_answer_item.xml

```diff
@@ -72,10 +72,9 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent">
 
-      <LinearLayout
-        android:orientation="horizontal"
-        android:layout_width="match_parent"
-        android:gravity="@{viewModel.hasConversationView ? Gravity.END : Gravity.START}"
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/container"
```

Comment:
Here & elsewhere: prefer not giving this a generic ID like container (does it actually need to have an ID at all?)

Reply:
I gave it an ID because that might be useful in the test case, but I think we can achieve that through some better way, so will remove it.